.travis.yml

```diff
@@ -33,6 +33,8 @@ install:
   # Print rustc and cargo versions
   - rustc -vV
   - cargo -vV
+  # Install wasm-gc for building substrate libs
+  - cargo install --git https://github.com/alexcrichton/wasm-gc --force
```

Comment:
I am not experienced with `wasm-gc` etc. but have you considered this?: https://github.com/alexcrichton/wasm-gc#wasm-gc

Reply:
I'm aware of that, but this was a prerequisite explicitly checked by the substrate `wasm-builder`. In any case the `substrate` deps are removed again (for now).